README.md

```diff
@@ -5,25 +5,26 @@
 
 The Graph command line interface.
 
-As of today, the command line interface consists of two commands:
+As of today, the command line interface consists of four commands:
 
 - `graph codegen` — generates TypeScript code for smart contract ABIs used in subgraphs.
 - `graph build` — compiles subgraphs to WebAssembly and deploys them to IPFS.
+- `graph deploy` — deploys subgraphs to a [Graph Node](https://github.com/graphprotocol/graph-node).
+- `graph remove` — removes subgraphs from a [Graph Node](https://github.com/graphprotocol/graph-node).
 
 ## How It Works
 
-`graph` takes a subgraph manifest (defaults to `subgraph.yaml`) with
+`graph` takes a subgraph manifest (defaults to `subgraph.yaml`) with references to
 
-- references to a GraphQL schema,
-- smart contract ABIs, and
-- mappings written in TypeScript/AssemblyScript,
+- a GraphQL schema,
+- smart contract ABIs
```

Comment:
Inconsistent commas here. Use a comma at the end of each bullet point and a full stop at the end of the last one.

Reply:
Damn, you're right. Yes, personally I'd start a new sentence after the last bullet point, but can't claim I'm consistent with that.

Reply:
I sometimes even use Oxford comma between the last two bullet points, like so:
```
- foo,
- bar, and
- baz.
```
I'm weird.